README.md

```diff
@@ -1,37 +1,39 @@
-Genome Diagram
----
+## Genome Diagram
 
-As lightweight Python 3 graphing API for genomic intervals and genomic features.
+A lightweight Python 3 graphing API for genomic intervals and genomic features.
 
 ```python
->>> import numpy as np
```

Comment:
All uncommented changes in the file are `>>>` removals.
IMHO the `>>>` get in the way under most circumstances.